flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/connector/kafka/sink/FlinkKafkaInternalProducer.java

```diff
@@ -85,16 +85,16 @@ public void beginTransaction() throws ProducerFencedException {
     public void abortTransaction() throws ProducerFencedException {
```

Comment:
Your commit message and PR title is wrong :)
`FlinkKafkaInternalProducerITCase`